### PR TITLE
RHIROS-456 fix inconsistent log format

### PR DESCRIPTION
**Pull request by r14chandra**
## Documentation update? :memo:

- [ ] Yes
- [X] No

## Security Checklist :lock:

Upon raising this PR please go through [RedHatInsights/secure-coding-checklist](https://github.com/RedHatInsights/secure-coding-checklist)

## :guardsman: Checklist :dart:

- [ ] Bugfix
- [ ] New Feature
- [X] Refactor
- [ ] Unittests Added
- [ ] DRY code
- [ ] Dependency Added
- [ ] DB Migration Added

## Additional :mega:

Feel free to add any other relevant details such as __links, notes, screenshots__, here.

### CODE REVIEW

**Review by saltgen — COMMENTED** [review]
Hi @r14chandra 
Is there any specific reason to move away from `fstrings`?

**Comment by r14chandra** [comment]
> Hi @r14chandra Is there any specific reason to move away from `fstrings`?

I see we have used `f strings` at 4 places and the rest of the places use `% operator`. To make it consistent, I replaced it with % operator.

**Comment by saltgen** [comment]
> > Hi @r14chandra Is there any specific reason to move away from `fstrings`?
> 
> I see we have used `f strings` at 4 places and the rest of the places use `% operator`. To make it consistent, I replaced it with % operator.

IIUC the format needs to be consistent. Here are a few reasons we should consider using `fstrings`,

- Readability
- Conciseness
- Better performance

**Comment by r14chandra** [comment]
> IIUC the format needs to be consistent. Here are a few reasons we should consider using `fstrings`, 
>     * Readability
>     * Conciseness
>     * Better performance

So, should prefer using `fstring` in all of the places?

**Comment by saltgen** [comment]
> > IIUC the format needs to be consistent. Here are a few reasons we should consider using `fstrings`,
> > * Readability
> > * Conciseness
> > * Better performance
> 
> So, should we prefer using `fstring` in all of the places?

That would be optimal, yes.

File: ros/api/common/utils.py [hunk 1]
```diff
@@ -64,10 +64,10 @@ def validate_request(*args, **kwargs):
         try:
             data = json.loads(request.data)
         except json.decoder.JSONDecodeError as err:
-            LOG.error('Decoding JSON has failed. %s', repr(err))
+            LOG.error(f"Decoding JSON has failed. {repr(err)}")
             abort(400, message="Decoding JSON has failed.")
         except TypeError as ex:
-            LOG.error('Invalid JSON format. %s', repr(ex))
+            LOG.error(f"Invalid JSON format. {repr(ex)}")
```
Comment (saltgen) [inline ros/api/common/utils.py:70]: Shall we add prefixes for this validator function? 
Not a problem if it is static, like `{REQUEST-VALIDATOR}`.

Or you could check in *args/**kwargs if there is a class object or `__name__` and then 
`{type(a).__name__}-VALIDATOR`

I intend to overload this and use for future endpoints.

**Comment by kgaikwad** [comment]
\retest

**Comment by ShivamPandya** [comment]
@r14chandra  Tests have been passed now.

**Pull request merged**